deploy/platform/kubernetes/Makefile.in

```diff
@@ -22,4 +22,4 @@ NAMESPACE ?= default
 AGENTLESS ?= false
 BACKEND_SERVICE ?= oap
 
-FEATURE_FLAGS ?= java-agent-injector,cluster,kubernetes-monitor,so11y,vm,als,event,istiod-monitor,satellite,rover,trace-profiling
+FEATURE_FLAGS ?= java-agent-injector,cluster,kubernetes-monitor,so11y,vm,als,event,istiod-monitor,satellite,rover,trace-profiling,mysql
```

Comment:
This is what the demo updater uses to install the env. So we don't need extra operations to update the demo to include this feature

Reply:
Then why `vm` is required?

Reply:
> Then why `vm` is required?

What do you mean by saying "required", this is more a choice of whether to enable a feature by default or not. I think this default setting is the same as our demo env so users can replicate the demo env with all default settings.

Reply:
Yes, MySQL is going to move on the demo env too.

Reply:
> Yes, MySQL is going to move on the demo env too.

Ok. So the change in this line is enough for the next update of demo env to include MySQL monitoring feature.

Reply:
I think you use this to control default deploymen for demo.skywalking.apache.org, right? https://github.com/SkyAPMTest/demo-env-installer/blob/7da2748916f5f4c2fa75c225a4ec7b3dd675d6f2/.github/workflows/update.yaml#L20

Reply:
Oh yes